cmd/ccli/ccli_test.go

```diff
@@ -129,6 +129,9 @@ func TestCompliance(t *testing.T) {
 
 			// Any unexpected error will be caught by being called directly on t from the fluent library.
 			tt.In.Fn(c, t)
+
+			// before moving on to the next test, explicitly flush the contents of the gRIBI server's RIB.
+			compliance.FlushServer(c, t)
```

Comment:
I think we should flush before each test, not after, since the t.Fatalf() above means we only get to flush if a test passes.

Reply:
Per our discussion off-line, I made these all be called in the tests as a deferred call - this allows order independent execution of the tests whilst also knowing whether to expect and need a flush to be successful.